### DISCUSSION TITLE
18

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#776] [Comments]
http://jojoldu.tistory.com/18

**Comment by jojoldu** [comment]
작성자: hwakyung

감사합니다 특히 bower 부분에서 시간적으로 큰 도움 받았네요!

**Comment by jojoldu** [comment]
작성자: 창천향로

도움이 되셨다니 다행입니다!!! 방문해주셔서 제가 더 감사드립니다^^

**Comment by jojoldu** [comment]
작성자: seoys

좋은정보 감사드립니다  :)

**Comment by jojoldu** [comment]
작성자: 창천향로

도움되셨다니 다행입니다^^ 방문해주셔서 감사합니다!

**Comment by chea-young** [comment]
제가 H10 오류가 나는데 node 프로젝트도 아니고  app.js 혹은 index.js를 사용하는 것도 아닌데 계속 오류가 나서 그런데 왜그런지 알 수 있을까요? 제 깃허브는 https://github.com/chea-young/locallibrary이거 입니다,
